gnome-extra/cinnamon/files/cinnamon-5.2.7-eds-detection.patch

```diff
@@ -0,0 +1,112 @@
+From ef463cc0aaedd714f2956daab227aeda1d87897e Mon Sep 17 00:00:00 2001
```

Comment:
Include links to where patches come from: https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto

Reply:
let me know when this bit is fixed